spoon

```diff
@@ -1,7 +1,15 @@
 #!/bin/bash
 #set -x
 
-IMAGE=tgmti/pdi
+# pull in any overrides to IMAGE from the .env file
+if [ -f .env ]
+then
+  export $(cat .env | sed 's/#.*//g' | xargs)
```

Comment:
"useless use of cat" -- `sed` can be passed `.env` as a target filename directly

I also don't think `xargs` is doing anything here. `$()` already flattens newlines into spaces.

Reply:
fixed